src/main/java/com/khoros/twitterapp/HomeFeedResource.java

```diff
@@ -0,0 +1,29 @@
+package com.khoros.twitterapp;
+
+import twitter4j.TwitterFactory;
+import twitter4j.Twitter;
+import twitter4j.Status;
+import twitter4j.TwitterException;
+import java.util.List;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/timeline")
+@Produces(MediaType.APPLICATION_JSON)
+public class HomeFeedResource {
+
+    @GET
+    public Response get() {
+        Twitter factory = new TwitterFactory().getSingleton();
+        try {
+            List<Status> tweetsFeed = factory.getHomeTimeline();
+            return Response.status(200).entity(tweetsFeed).type("application/json").build();
```

Comment:
type should come from the @Produces up above

Reply:
It gave me some kind of MediaParser error due to it interpreting the data as Application/Octet or something like that. I added it for that reason.

Reply:
I didn't get the error again, so I updated it anyway